include/deal.II/base/derivative_form.h

```diff
@@ -466,6 +466,29 @@ apply_transformation(const DerivativeForm<1, dim, spacedim, Number1> &grad_F,
 
 
 
+/**
+ * Similar to the previous apply_transformation().
```

Comment:
```suggestion
 * Similar to the previous apply_transformation(), specialized for the case `dim == spacedim`
 * where we can return a rank-2 tensor instead of the more general `DerivativeForm`.
```